redaxo/src/addons/structure/plugins/content/fragments/slice_select.php

```diff
@@ -0,0 +1,3 @@
+<?php
+
```

Comment:
macht es ggf. sinn das wir hier mit phpdoc kurz "deklarieren" welche variablen es gibt, bspw:

```suggestion
/**
 * @var int $id the module id
 * @var string $title the module name
 * @var string $href the module url
 */
```

o.ä.

Reply:
Habs mal angepasst